### PR TITLE
XCreds 3.2 update

### PR DESCRIPTION
**Pull request by twocanoes**
New keys for XCreds version 3.2

### CODE REVIEW

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist [hunk 1]
```diff
@@ -441,7 +441,59 @@ Note that Google does not support the offline_access scope so instead use the pr
...
+			<key>pfm_name</key>
+			<string>ropgClientSecret</string>
+			<key>pfm_title</key>
+			<string>ROPG Client Secret</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<string>0</string>
```
Comment (relgit) [inline Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:496]: The type of the default value should match the preference key and be Integer.

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist [hunk 2]
```diff
@@ -441,7 +441,59 @@ Note that Google does not support the offline_access scope so instead use the pr
...
+			<key>pfm_name</key>
+			<string>shouldVerifyPasswordWithRopg</string>
+			<key>pfm_title</key>
+			<string>Use ROPG when testing password</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>ROPG Client ID for use when checking password</string>
```
Comment (relgit) [inline Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:472]: Looking at the manifest, most descriptions end with a period. Let's add one here too?

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist [hunk 3]
```diff
@@ -441,7 +441,59 @@ Note that Google does not support the offline_access scope so instead use the pr
...
+			<key>pfm_name</key>
+			<string>ropgClientID</string>
+			<key>pfm_title</key>
+			<string>ROPG Client ID</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>ROPG Client Secret for use when checking password</string>
```
Comment (relgit) [inline Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:484]: This should end with a period, like in the comment above.

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist [hunk 4]
```diff
@@ -709,6 +761,20 @@ Note that Google does not support the offline_access scope so instead use the pr
...
+			<key>pfm_default</key>
+			<false/>
+			<key>pfm_description</key>
+			<string>Check if network is up. If not, select username and password login window.</string>
+			<key>pfm_documentation_url</key>
+			<string>https://twocanoes.com/knowledge-base/xcreds-admin-guide/#preferences</string>
+			<key>pfm_name</key>
+			<string>shouldDetectNetworkToDetermineLoginWindow</string>
+			<key>pfm_title</key>
+			<string>Should Detect Network ToDetermine LoginWindow</string>
```
Comment (relgit) [inline Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:774]: Two spaces sneaked out of where they're supposed to be.

**Review by relgit — CHANGES_REQUESTED** [review]
Thanks for the PR @twocanoes. Please have a look at the comments I've added.

**Comment by twocanoes** [comment]
Fix items requested in previous PR

**Pull request merged**